packages/ocaml-variants/ocaml-variants.4.06.1+trunk+safe-string/opam

```diff
@@ -11,14 +11,14 @@ conflict-class: "ocaml-core-compiler"
 flags: compiler
 setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
 build: [
-  ["./configure" "-prefix" prefix "-with-debug-runtime" "-safe-string"]
```

Comment:
This switch is uninstallable because the ```-safe-string``` option doesn't exists. Could you rename it to make it consistent with the stable variants (i.e. ```4.06.1+trunk+force-safe-string```). Also ```4.06.1+trunk+default-unsafe-string``` is missing but I don't know if it change something at this point.